src/gridstack.ts

```diff
@@ -2530,7 +2526,6 @@ export class GridStack {
     } else if (node._isExternal) {
       // item came from outside (like a toolbar) so nuke any node info
       delete node.el;
-      delete el.gridstackNode;

```

Comment:
doesn't make sense to delete one and not the other... will have to debug the code for that particular case and make sure it doesn't break other cases..

Reply:
I agree. I noticed errors in the console and problems with blocks not being deleted after dragging. Restored this function.